### PR TITLE
Little fix / improvement of gtkIcons detection

### PR DESCRIPTION
**Pull request by Flashwalker**
* Add  **gtkIcons** detection from **$HOME/.gtkrc-2.0.mine** file.
* Fix  **gtkIcons** detection from **${HOME}/.config}/rox.sourceforge.net/ROX-
Filer/Options** file.

### CODE REVIEW

File: screenfetch-dev [hunk 1]
```diff
@@ -2002,16 +2002,17 @@ detectgtk () {
 					fi
 				fi
 				# $HOME/.gtkrc.mine theme detect only
-				if [ -f "$HOME/.gtkrc.mine" ]; then
-					if grep -q "^include" $HOME/.gtkrc.mine; then
-						gtk2Theme=$(grep '^include.*gtkrc' $HOME/.gtkrc.mine | awk -F "/" '{ print $5 }')
+				if [[ -f "$HOME/.gtkrc.mine" || -f "$HOME/.gtkrc-2.0.mine" ]]; then
```
Comment (darealshinji) [inline screenfetch-dev]: If this line returns false, the variable `$MINEGTKRC` will not be set and can't be used in line 2014/2015.
You should also use lowercase letters, as uppercase letters are normally used by environment variables (`$CFLAGS`, `$HOME`, `$LD_LIBRARY_PATH`, ...).

Comment (Flashwalker) [inline screenfetch-dev]: Fixed.

**Comment by darealshinji** [comment]
I think this is better: https://github.com/darealshinji/screenFetch/commit/6c03d9f014acf837349b41eb4cde91cb874f5eb3

**Comment by KittyKatt** [comment]
Will come back and compare. Thanks for all the hard work lately @darealshinji

**Comment by darealshinji** [comment]
Changes added manually with https://github.com/KittyKatt/screenFetch/pull/348

**Pull request closed without merging**